tests/test_config.py

```diff
@@ -72,9 +72,9 @@ def test_parse_json(self, tmp_path, spec_list):
         # a little bit of alternative field name testing
         for spec in spec_list:
             spec_dict = spec._asdict()
-            spec_dict["filename"] = spec_dict.pop("path_spec")
+            spec_dict["Filename"] = spec_dict.pop("path_spec")
             if "license_type" in spec_dict:
-                spec_dict["usage"] = spec_dict.pop("license_type")
+                spec_dict["Usage Rights"] = spec_dict.pop("license_type")
```

Comment:
Kind of hilarious that in addition to not testing case-sensitivity, I didn't test the key -> key_name permutations.